Formula/zion.rb

```diff
@@ -0,0 +1,26 @@
+class Zion < Formula
+  desc "Provides E2EE gateway to create secure communication"
+  homepage "https://nullby1e.github.io/zion/"
+  url "https://github.com/a1eaiactaest/zion-formula/archive/0.0.2.tar.gz"
```

Comment:
It should be downloading from the original upstream source, not just a tarbal in some repo.

Reply:
well, the tarball is only [configure](https://github.com/a1eaiactaest/zion-formula/blob/master/configure) script, check it. all of the files are downloaded from original source.

Reply:
That means it's going against https://docs.brew.sh/Acceptable-Formulae#we-dont-like-install-scripts-that-download-unversioned-things and https://docs.brew.sh/Acceptable-Formulae#stable-versions

Reply:
Is there a way to do the same (connect with proxy and only then download original source) in a formula file?

Reply:
No, if the source is not publicly available brew won't be able to fetch it.

Reply:
It is public except that tor is needed to download a file. which can be done with proxy on curl or other tool.

Reply:
Even if we manage to overcome the Tor hurdle, it's still an unversioned download.